OnServiceChange/handler.ts

```diff
@@ -236,10 +281,15 @@ export const createServiceChangeHandler = (
     RA.map(validateDocument),
     RA.map(
       E.fold(
-        onInvalidDocument,
+        document => onInvalidDocument(document, telemetryClient),
```

Comment:
You can define `onInvalidDocument` and all the others as high order functions, passing telemetryClient as first input.

Reply:
Oh, now you remember High Order Functions 😄 
Yes, thanks. Fix with 8827d28

Reply:
;)